.gitignore

```diff
@@ -21,3 +21,11 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# generated css and css.map files
+.sass-cache/
+*.css.map
+*.sass.map
+*.scss.map
```

Comment:
possible conflict with our approach to scss

Reply:
Done!